src/model/trust.model.ts

```diff
@@ -48,7 +48,7 @@ export interface TrustReviewStatus {
 export interface TrustIndividual {
   id?: string;
   ch_references?: string;
-  type: RoleWithinTrustType;
+  type: RoleWithinTrustType | undefined;
```

Comment:
Just wondering if the OE-API accepts the type if it is undefined, have we tried saving the data to mongo if the type gets set to undefined for the 5001 case.

Reply:
i'll try and replicate this in docker, just wondering what will happen if we submit type as undefined to the SDK/API

Reply:
Have tested with a full e2e web journey. The type is omitted from the record in Mongo if it is undefined. This works fine for subsequent pages (e.g Check Your Answers). 
Would also need to see if there are no issues when the data is sent to CHIPS.